packages/mdc-list/foundation.ts

```diff
@@ -304,6 +304,20 @@ export class MDCListFoundation extends MDCFoundation<MDCListAdapter> {
     return lastIndex;
   }
 
+  /**
+   * @param itemIndex Index of the list item
+   * @param isEnabled Sets the list item to enabled or disabled.
+   */
+  setEnabled(itemIndex: number, isEnabled: boolean): void {
+    if (isEnabled) {
```

Comment:
Can you also add validation for itemIndex?

See `setSelectedIndex` method for example.

Reply:
Done